master/buildbot/test/unit/reporters/test_base.py

```diff
@@ -189,19 +189,19 @@ def generate_throw(*args, **kwargs):
 
     @defer.inlineCallbacks
     def test_reports_sent_in_order_despite_slow_generator(self):
-        gen = self.setup_mock_generator([('fake1', None, None)])
+        gen = self.setup_mock_generator([('builds', None, None)])
```

Comment:
I think we don't need to change this test to not create an illusion that the test depends on real event keys. Was the test broken for some reason?

Reply:
Nevermind, I was typing before thinking.